cmd/av/auth_status.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/aviator-co/av/internal/gql"
+	"github.com/shurcooL/graphql"
+	"github.com/spf13/cobra"
+)
+
+var authStatusCmd = &cobra.Command{
+	Use:          "status",
+	Short:        "check auth status",
+	SilenceUsage: true,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		client := gql.GraphQLClient()
+
+		var query struct {
+			Viewer struct {
+				Email graphql.String
+			}
+		}
+
+		err := client.Query(context.Background(), &query, nil)
+		if err != nil {
+			_, _ = fmt.Fprintln(os.Stderr, err)
+			fmt.Printf("<ERROR: %v>\n", err)
+			return err
+		}
+
+		if query.Viewer.Email == "" {
+			fmt.Fprintln(
+				os.Stderr,
+				"Error: Could not find auth info, please verify that your API Token is correct.",
+			)
+			return nil
```

Comment:
You can return `ErrExitSilently{1}` here that way the process exits with a non-zero exit code.